pkg/security/agent/agent.go

```diff
@@ -162,6 +162,8 @@ func (rsa *RuntimeSecurityAgent) GetStatus() map[string]interface{} {
 				}
 				base["environment"] = environment
 			}
+			base["selfTestsNumberOfSuccess"] = cfStatus.SelfTestsNumberOfSuccess
```

Comment:
It would be interesting to know which test succeeded or failed.
What about something like
`
...
"selfTests": {
    "lastTime": "...",
    "results": {
        "chown": "success",
        "open": "failed",
    }
}
`

Reply:
Indeed! It should be nicer with the last commit :)